bin/gem_installer

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
```

Comment:
This script is basically a copy of `bin/gem_downloader`... The only differences I can see are:
- this doesn't support `:fetch`,
- this doesn't use `--explain` in the `gem install` command.

Can we please add some comments on why we need both scripts?

Reply:
Because packaging on Linux and Windows still works differently. The Linux one doesn't seem to install the gems, hence the `--explain`. I can add a `--install` flag to the downloader and the feature, is that what you're asking me to do?

Reply:
The script is now different, without dependencies on curl, and stands on its own. Note that I copied the script from omnibus-tb and preserved the naming there, so I don't see a reason to change it.

Reply:
The script is still 90% similar.

That said, what's "omnibus-tb"? Do you mean treasure-data/omnibus-td-agent? If so, would it not make sense to copy a (modified) script from our fork, for consistency, rather than a (potentially diverged) script from the upstream repo?

In any case, I will send a follow-up PR to unify the scripts, so we can leave it as-is in this PR.

Reply:
Yes, I meant `treasure-data/omnibus-td-agent`, I'm sorry for confusing you. Thanks for all the ideas, I think this is fine.